scripts/tools/zap/generate.py

```diff
@@ -149,9 +148,7 @@ def extractGeneratedIdl(output_dir, zap_config_path):
 
 
 def runGeneration(zap_file, zcl_file, templates_file, output_dir):
-    generator_dir = os.getenv('ZAP_PATH', getDirPath('third_party/zap/repo'))
-    os.chdir(generator_dir)
-    subprocess.check_call(['node', './src-script/zap-generate.js', '-z',
+    subprocess.check_call(['zap-cli', 'generate', '-z',
```

Comment:
So in practical terms, how does local development work now?  When one needs to rerun the zap codegen, i.e. run this script, how does one do that?

Reply:
In particular, perhaps we should keep checking for ZAP_PATH and if set move there, and perhaps we should check for whether zap-cli is in our PATH and produce a useful error explaining what to do if it's not?

Most importantly: it _must_ be clear how to run this script and have it use a _locally modified_ ZAP.

Reply:
Just running the script seems fine: zap-cli is expected to be installed.

Reply:
Updated to support zap in devel, install or $PATH after slack conversation. I also now print some explanation of what went wrong.